Libraries/LibWeb/WebAssembly/Memory.cpp

```diff
@@ -21,6 +22,11 @@ WebIDL::ExceptionOr<GC::Ref<Memory>> Memory::construct_impl(JS::Realm& realm, Me
 {
     auto& vm = realm.vm();
 
+    // https://github.com/WebAssembly/threads/blob/main/proposals/threads/Overview.md#webassemblymemory-constructor
```

Comment:
These references should be to the proposal's spec, rather than its explainer.

e.g. https://webassembly.github.io/threads/js-api/index.html#dom-memory-memory

Reply:
Thanks! The spec is indeed much easier to use :)

Fixed.